index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet"/>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav-bar">
+            <a href=""><i class="fa fa-bars hamburger" aria-hidden="true"></i></a>
+        </nav>
+    </header>
+    <section class="section-1">
+        <h1 class="heading-1">
+            Busiwa Liuma
+        </h1>
+        <p class="par-1">
+        Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+        If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </p>
+        <button class="button-1" type="button">Start Collaboration</button>
```

Comment:
- [ ] There should be a list of icons on your page and the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Please fix that.
- [ ] Please you have to make a professional readme file. For now your portfolio has not been done yet. Try to fix that.

Reply:
Thanks it's been handled